BackEnd/src/business/games.ts

```diff
@@ -51,17 +45,12 @@ export async function SaveDataByMatchId(matchId: string, updatePlayerStats: bool
     seasonId = (await GetDbCode(gameData.info.tournamentCode))?.seasonId;
   }
 
-  if (seasonId && updatePlayerStats) {
+  let recentlyBuiltRisenTeams = await hasRecentlyBuiltRisenTeams(seasonId);
+  if (seasonId && !recentlyBuiltRisenTeams) {
     await buildRisenTeams(seasonId);
   }
 
-  const savedGameModel: GameModel = await SaveSingleMatchById(matchId, gameData, seasonId);
-
-  if (updatePlayerStats) {
-    await updatePlayerStatsForGame(matchId);
-  }
```

Comment:
Seems like there is no replacement for this. So player stats are no longer getting updated on game save

Reply:
L28 has the replacement